docs/installation/_index.md

```diff
@@ -52,69 +41,129 @@ If you are using a single board computer (SBC) or a microcontroller, prepare you
-   Download your robot's config file from the **Setup** tab of your robot on the Viam app. </li>
+By default, `viam-server` will start automatically when your system boots, but you can [change this behavior](/installation/manage/) if desired.
 
-<!-- The below has to be in HTML because we're using a table inside another table with indentation-->
-<ol start="2">
-<li> <strong>Download and install <code>viam-server</code>.</strong>
+## Install on a macOS or Linux computer
+
+You can also install `viam-server` on a macOS or Linux computer.
+This is useful if you don't have a board available, or if you want to run a Viam [service](/services/) with your robot that requires more computing resources than is otherwise available on an SBC.
+Select the tab below for your operating system:
+
+{{< tabs name="Install on computer" >}}
+{{% tab name="macOS computer" %}}
+
+### Install on a macOS computer
+
+`viam-server` is available for macOS users through [Homebrew](https://docs.brew.sh/Installation), and supports both Intel and Apple Silicon macOS computers.
+To install `viam-server` on a macOS computer:
+
+1. Go to the [Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot).
+   If this is your first time using the Viam app, you must create an account first.
+
+1. On the **Setup** tab, select `Mac` for **Mode**.
+
+1. Follow the steps shown on the **Setup** tab to install `viam-server` on your macOS computer.
+
+1. Once `viam-server` is installed and running, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your computer has successfully connected.
+
+{{% /tab %}}
+{{% tab name="Linux computer" %}}
+
+### Install on a Linux computer
+
+`viam-server` is distributed for Linux as an [AppImage](https://appimage.org/).
+The AppImage is a single, self-contained binary that runs on 64-bit Linux systems running the `aarch64` or `x86_64` architectures, with no need to install any dependencies.
+To install `viam-server` on a Linux computer:
+
+1. Go to the [Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot).
+   If this is your first time using the Viam app, you must create an account first.
+
+1. On the **Setup** tab, select `Linux` for **Mode** and select the appropriate **Architecture** for your computer.
+   On most Linux operating systems, you can run `uname -m` to confirm your computer's architecture.
+
+1. Follow the steps shown on the **Setup** tab to install `viam-server` on your Linux computer.
+
+1. Once `viam-server` is installed and running, return to the **Setup** page on the [Viam app](https://app.viam.com) and wait for confirmation that your computer has successfully connected.
+
+By default, `viam-server` will start automatically when your system boots, but you can [change this behavior](/installation/manage/) if desired.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Local installation
```

Comment:
Let's discuss this on slack

Reply:
@npentrel I applied some small languages changes based on our discussion. Let me know if you'd like to see any addition changes. 👍